### ISSUE TITLE
Integeration of WBOT with other systems

### ISSUE THREAD

**Issue by alijundi** [vasani-arpit/WBOT#53]
Dear Mr. Vasani,

Is it possible to get the source code of this amazing project. I really need to integrate it with another system in order to generate responses from database base on the phone numbers and user selection.

Thank you for your valuable project!

**Comment by vasani-arpit** [comment]
Yes. Head over to code tab in GitHub to get the source code. you can also get source code of each specific release from this page https://github.com/vasani-arpit/WBOT/releases

This can be integrated with any other system easily. That part of the project is work-in-progress. You can see the progress in webhook branch of the code.

For more questions you can contact me on my [email](https://github.com/vasani-arpit/WBOT#-contact-me-)

**Comment by alijundi** [comment]
Dear @vasani-arpit I download the source code. Can you please let me know what IDE did you use in generating the WBOT. I am thinking I should use Visual Studio but I am not sure! 

Thank you in advance.

**Comment by vasani-arpit** [comment]
https://code.visualstudio.com/

**Comment by alijundi** [comment]
Thanks @vasani-arpit. I was able to download VScode and run the source code. I have only one question I am trying to debug the code to better understand how it works. It seems that the WAPI.js and inject.js are added using the below code:

            var filepath = path.join(__dirname, "WAPI.js");
            await page.addScriptTag({ path: require.resolve(filepath) });
            filepath = path.join(__dirname, "inject.js");
            await page.addScriptTag({ path: require.resolve(filepath) });"

and therefore the breakpoints are not getting hit! I also checked the loaded scripts using the left side panel and noticed that the scripts are not loaded!

I appreciate your guidance on how to debug the code in order to be able to integrate it with other systems, etc.
